NRzure/dashboards/example-dashboard/Intern Hackathon Azure.json

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
Let's also rename this dashboard file/directory to something more descriptive. Instead of `dashboards/example-dashboard` let's get it in a directory like this `dashboards/dashboard-name`